Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj

```diff
@@ -12,4 +13,31 @@
     <ProjectReference Include="..\DotSpatial.Symbology\DotSpatial.Symbology.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Why did you add those lines?
VS 2022 shows this information without having to add it to the csproj file.